template/.nvmrc

```diff
@@ -0,0 +1 @@
+18.12
```

Comment:
@jstorm31 did you try this? It seems like `madge` has `detective-postcss` dependency which requires at most node 16...

Reply:
https://github.com/pahen/madge/issues/329 :/

Reply:
Sad story, downgraded back to version `16.19`